packages/test/test-service-load/src/gcDataStores.ts

```diff
@@ -92,15 +92,17 @@ const BlobActivityType = {
 type BlobActivityType = typeof BlobActivityType[keyof typeof BlobActivityType];
 
 /**
- * Activities that can be performed by the leaf blob object.
+ * Activities that can be performed by the leaf data object.
  */
 const LeafActivityType = {
 	/** Don't do anything. */
 	None: 0,
```

Comment:
Add anotherNone to give a little more weight to it to get more incremental coverage?

Reply:
It's 33% already which is pretty good right. The last one `Count` is not an activity.